digdag-tests/src/test/resources/acceptance/td/td/td_result_url.dig

```diff
@@ -0,0 +1,8 @@
+timezone: UTC
+
++run:
+  td>: query.sql
+  result_url: td://${secret:td.apikey}@/${td.database}/result_url
```

Comment:
If the placeholder isn't converted at all or is replaced with empty string, this job should fail. So this test can check if the changes work well.
👍